backend/endpoints/rooms.py

```diff
@@ -127,7 +127,7 @@ async def remove_member(
             detail="user not a member of the room",
         )
 
-    if admin_id not in room_data["room_members"]:
+    if admin_id is not None and admin_id not in room_data["room_members"]:
         raise HTTPException(
             status_code=status.HTTP_404_NOT_FOUND,
             detail="admin id specified not a member of the room",
```

Comment:
I tested with this room (`61e72d1265934b58b8e5d220`, Channel).
The room had only a member who is the admin, `61e59f1b65934b58b8e5d1cb`.
I was able to remove the admin by passing the admin id to both **member_id** and **admin_id** parameters.
Now, the room has no members as the admin has been removed.

This shouldn't be possible, right? @kinglighthill